test/bxml-responses/hangup.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Response>
+     <Hangup/>
```

Comment:
That's a standard empty-element tag in XML - should work. I can make it `<hangup></hangup>` though